<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PkmlpCounter</class>
 <widget class="QMainWindow" name="PkmlpCounter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1783</width>
    <height>1334</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TallyCounter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>55</number>
    </property>
    <property name="rightMargin">
     <number>55</number>
    </property>
    <property name="bottomMargin">
     <number>22</number>
    </property>
    <item row="1" column="0" colspan="3">
     <widget class="QGroupBox" name="grpCounterDisplay">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 2px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 30px;
    padding: -15 3px 0 3px;
}</string>
      </property>
      <property name="title">
       <string>Console</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="_2">
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="topMargin">
        <number>55</number>
       </property>
       <property name="rightMargin">
        <number>30</number>
       </property>
       <property name="bottomMargin">
        <number>30</number>
       </property>
       <property name="verticalSpacing">
        <number>15</number>
       </property>
       <item row="0" column="4">
        <widget class="QPushButton" name="btnAdd">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>23</pointsize>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">Text-align:left</string>
         </property>
         <property name="text">
          <string> Add</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>arrow_up.png</normaloff>arrow_up.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>75</width>
           <height>75</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="3">
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3" rowspan="3">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="btnReset">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>23</pointsize>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">Text-align:left</string>
         </property>
         <property name="text">
          <string> Reset</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>null.png</normaloff>null.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="3">
        <widget class="QProgressBar" name="progressBarCounter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="value">
          <number>24</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="format">
          <string>%p%</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QPushButton" name="btnSubtract">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>23</pointsize>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">Text-align:left</string>
         </property>
         <property name="text">
          <string> Subtract</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>arrow_down.png</normaloff>arrow_down.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>75</width>
           <height>75</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="3">
        <widget class="QLCDNumber" name="lcdNumberCounter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="digitCount">
          <number>3</number>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Filled</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="grpCounterType">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 2px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 30px;
    padding: -15 3px 0 3px;
}</string>
      </property>
      <property name="title">
       <string>Type</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>30</number>
       </property>
       <item row="0" column="0">
        <widget class="QRadioButton" name="radioButtonRollover">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Rollover</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="radioButtonStopping">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Stopping</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QLabel" name="lblFusszeile">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>(c) www.pkmlp.ch  -  Peter Kessler</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="2">
     <widget class="QGroupBox" name="grpCounterSettings">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox {
    border: 2px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 30px;
    padding: -15 3px 0 3px;
}</string>
      </property>
      <property name="title">
       <string>Settings</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="topMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>30</number>
       </property>
       <item row="5" column="0" colspan="3">
        <widget class="QSlider" name="horsliderSteps">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>5</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="sliderPosition">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lbl_1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>1</string>
         </property>
         <property name="indent">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QComboBox" name="comboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="lbl_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>3</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QLabel" name="lblSchrittweite">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>15</width>
           <height>55</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Schrittweite</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QLabel" name="lbl_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>5</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="indent">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="lblStellen">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>55</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>55</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Stellen</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="lblTitel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>150</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>31</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TallyCounter</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1783</width>
     <height>56</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="statusTip">
    <string>Main Menu</string>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuApp">
    <property name="statusTip">
     <string>App</string>
    </property>
    <property name="title">
     <string>App</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuConsole">
    <property name="statusTip">
     <string>Console</string>
    </property>
    <property name="title">
     <string>Console</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionReset"/>
    <addaction name="actionSubtract"/>
   </widget>
   <widget class="QMenu" name="menuType">
    <property name="toolTip">
     <string/>
    </property>
    <property name="statusTip">
     <string>Type</string>
    </property>
    <property name="title">
     <string>Type</string>
    </property>
    <addaction name="actionRollover"/>
    <addaction name="actionStopping"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="statusTip">
     <string>Settings</string>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuStellen">
     <property name="statusTip">
      <string>Stellen</string>
     </property>
     <property name="title">
      <string>Stellen</string>
     </property>
     <addaction name="action_1_stellig"/>
     <addaction name="action_2_stellig"/>
     <addaction name="action_3_stellig"/>
    </widget>
    <widget class="QMenu" name="menuSchrittweite">
     <property name="statusTip">
      <string>Schrittweite</string>
     </property>
     <property name="title">
      <string>Schrittweite</string>
     </property>
     <addaction name="action_1"/>
     <addaction name="action_2"/>
     <addaction name="action_3"/>
     <addaction name="action_4"/>
     <addaction name="action_5"/>
    </widget>
    <addaction name="menuStellen"/>
    <addaction name="menuSchrittweite"/>
   </widget>
   <addaction name="menuApp"/>
   <addaction name="menuConsole"/>
   <addaction name="menuType"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
   <property name="statusTip">
    <string>Add</string>
   </property>
  </action>
  <action name="actionSubtract">
   <property name="text">
    <string>Subtract</string>
   </property>
   <property name="statusTip">
    <string>Subtract</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="statusTip">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>About</string>
   </property>
  </action>
  <action name="actionRollover">
   <property name="text">
    <string>Rollover</string>
   </property>
   <property name="statusTip">
    <string>Rollover</string>
   </property>
  </action>
  <action name="actionStopping">
   <property name="text">
    <string>Stopping</string>
   </property>
   <property name="statusTip">
    <string>Stopping</string>
   </property>
  </action>
  <action name="action_1_stellig">
   <property name="text">
    <string>1-stellig (0-9)</string>
   </property>
   <property name="toolTip">
    <string>1-stellig (0-9)</string>
   </property>
   <property name="statusTip">
    <string>1-stellig (0-9)</string>
   </property>
  </action>
  <action name="action_2_stellig">
   <property name="text">
    <string>2-stellig (0-99)</string>
   </property>
   <property name="toolTip">
    <string>2-stellig (0-99)</string>
   </property>
   <property name="statusTip">
    <string>2-stellig (0-99)</string>
   </property>
  </action>
  <action name="action_3_stellig">
   <property name="text">
    <string>3-stellig (0-999)</string>
   </property>
   <property name="toolTip">
    <string>3-stellig (0-999)</string>
   </property>
   <property name="statusTip">
    <string>3-stellig (0-999)</string>
   </property>
  </action>
  <action name="action_1">
   <property name="text">
    <string> +/- 1</string>
   </property>
   <property name="statusTip">
    <string>Schrittweite: +/- 1</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string> +/- 2</string>
   </property>
   <property name="statusTip">
    <string>Schrittweite: +/- 2</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string> +/- 3</string>
   </property>
   <property name="statusTip">
    <string>Schrittweite: +/- 3</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string> +/- 4</string>
   </property>
   <property name="statusTip">
    <string>Schrittweite: +/- 4</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string> +/- 5</string>
   </property>
   <property name="statusTip">
    <string>Schrittweite: +/- 5</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../QT-Projekte/pkmlpCounter_99/images/images.qrc"/>
 </resources>
 <connections/>
</ui>
